share/wake/lib/system/job.wake

```diff
@@ -86,23 +89,40 @@ export data Persistence =
 
 # A Plan describes a not-yet-executed Job
 tuple Plan =
-  export Label:        String       # The label used when showing the command during execution
-  export Command:      List String  # The command-line arguments (the first is the command to run)
-  export Visible:      List Path    # Only these files should be available to the command
-  export Environment:  List String  # KEY=VALUE environment variables fed to the command
-  export Directory:    String       # The directory in which the command should be run
-  export Stdin:        String       # The file to which standard input should be connected
-  export Stdout:       LogLevel     # How should standard output be displayed during a build
-  export Stderr:       LogLevel     # How should standard error be displayed during a build
-  export Echo:         LogLevel     # Echo the command to this stream
-  export Persistence:  Persistence  # See Persistence table above
-  export LocalOnly:    Boolean      # Must run directly in the local workspace; no output detection performed
-  export Resources:    List String  # The resources a runner must provide to the job (licenses/etc)
-  export RunnerFilter: Runner => Boolean # Reject from consideration Runners which the Plan deems inappropriate
-  export Usage:        Usage        # User-supplied usage prediction; overruled by database statistics (if any)
-  export FnInputs:     (List String => List String) # Modify the Runner's reported inputs  (files read)
-  export FnOutputs:    (List String => List String) # Modify the Runner's reported outputs (files created)
-  export IsAtty:       Boolean      # Should job run in psuedoterminal
+  # The label used when showing the command during execution
+  export Label:        String
+  # The command-line arguments (the first is the command to run)
+  export Command:      List String
+  # Only these files should be available to the command
+  export Visible:      List Path
+  # KEY=VALUE environment variables fed to the command
+  export Environment:  List String
+  # The directory in which the command should be run
+  export Directory:    String
+  # The file to which standard input should be connected
```

Comment:
Might note that the empty string is interpreted as there being no Stdin file

Reply:
[Clarified here](https://github.com/sifive/wake/pull/1206/commits/1fc38d4549433709dd823c8af2fd16751eb68308)